src/kbmod/masking.py

```diff
@@ -2,6 +2,44 @@
 """
 
 import kbmod.search as kb
+from kbmod.search import RawImage
+
+
+def mask_trajectory(trj, stack, r, bitmask=1):
```

Comment:
You can remove bitmask from the arguments.

Reply:
Removed bitmask